contracts/external/gmxV2/GmxRegistryV2.sol

```diff
@@ -0,0 +1,289 @@
+import { IGmxV2IsolationModeWrapperTraderV2 } from "../interfaces/gmx/IGmxV2IsolationModeWrapperTraderV2.sol";
+import { IGmxWithdrawalHandler } from "../interfaces/gmx/IGmxWithdrawalHandler.sol";
+
+/**
+ * @title   GmxRegistryV2
+ * @author  Dolomite
+ *
+ * @notice  Implementation for a registry that contains all of the GMX V2-related addresses. This registry is needed to
+ *          offer uniform access to addresses in an effort to keep Dolomite's contracts as up-to-date as possible
+ *          without having to deprecate the system and force users to migrate (losing any vesting rewards / multiplier
+ *          points) when Dolomite needs to point to new contracts or functions that GMX introduces.
+ */
+contract GmxRegistryV2 is IGmxRegistryV2, BaseRegistry {
+    // ==================== Constants ====================
+
+    bytes32 private constant _FILE = "GmxRegistryV2";
+
+    // solhint-disable max-line-length
+    bytes32 private constant _ETH_USD_MARKET_TOKEN_SLOT = bytes32(uint256(keccak256("eip1967.proxy.ethUsdMarketToken")) - 1);
+    bytes32 private constant _GMX_DEPOSIT_HANDLER_SLOT = bytes32(uint256(keccak256("eip1967.proxy.gmxDepositHandler")) - 1);
+    bytes32 private constant _GMX_DEPOSIT_VAULT_SLOT = bytes32(uint256(keccak256("eip1967.proxy.gmxDepositVault")) - 1);
+    bytes32 private constant _GMX_EXCHANGE_ROUTER_SLOT = bytes32(uint256(keccak256("eip1967.proxy.gmxExchangeRouter")) - 1);
+    bytes32 private constant _GMX_READER_SLOT = bytes32(uint256(keccak256("eip1967.proxy.gmxReader")) - 1);
+    bytes32 private constant _GMX_ROUTER_SLOT = bytes32(uint256(keccak256("eip1967.proxy.gmxRouter")) - 1);
+    bytes32 private constant _GMX_WITHDRAWAL_HANDLER_SLOT = bytes32(uint256(keccak256("eip1967.proxy.gmxWithdrawalHandler")) - 1);
+    bytes32 private constant _GMX_V2_UNWRAPPER_TRADER_SLOT = bytes32(uint256(keccak256("eip1967.proxy.gmxV2UnwrapperTrader")) - 1);
+    bytes32 private constant _GMX_V2_WRAPPER_TRADER_SLOT = bytes32(uint256(keccak256("eip1967.proxy.gmxV2WrapperTrader")) - 1);
+    // solhint-enable max-line-length
+
+    // ==================== Initializer ====================
+
+    function initialize(
+        address _gmxExchangeRouter,
+        address _gmxReader,
+        address _gmxRouter,
+        address _gmxDepositHandler,
+        address _gmxDepositVault,
+        address _gmxWithdrawalHandler,
+        address _ethUsdMarketToken,
+        address _dolomiteRegistry
+    ) external initializer {
+        _ownerSetGmxExchangeRouter(_gmxExchangeRouter);
+        _ownerSetGmxReader(_gmxReader);
+        _ownerSetGmxRouter(_gmxRouter);
+        _ownerSetEthUsdMarketToken(_ethUsdMarketToken);
+        _ownerSetGmxDepositHandler(_gmxDepositHandler);
+        _ownerSetGmxDepositVault(_gmxDepositVault);
+        _ownerSetGmxWithdrawalHandler(_gmxWithdrawalHandler);
+
+        _ownerSetDolomiteRegistry(_dolomiteRegistry);
+    }
+
+    // ==================== Functions ====================
```

Comment:
Add a function for initializing the wrapper/unwrapper params. See jUSDC for an example